### PR TITLE
chore: BSC PancakeSwap forks VIP, PancakeSwapV2, Saddle BTC pool, Curve gas schedule [TKR-41] [TKR-11] [TKR-79] [TKR-86]

### PR DESCRIPTION
**Pull request by romainbutteaud**
## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: contracts/zero-ex/contracts/src/features/PancakeSwapFeature.sol [hunk 1]
```diff
@@ -356,11 +374,31 @@ contract PancakeSwapFeature is
...
                         mstore(0xB00, FF_SUSHISWAP_FACTORY)
                         mstore(0xB15, salt)
                         mstore(0xB35, SUSHISWAP_PAIR_INIT_CODE_HASH)
                     }
+                    case 3 {
+                        mstore(0xB00, FF_APESWAP_FACTORY)
+                        mstore(0xB15, salt)
+                        mstore(0xB35, APESWAP_PAIR_INIT_CODE_HASH)
+                    }
+                    case 5 {
```
Comment (dorothy-zbornak) [inline contracts/zero-ex/contracts/src/features/PancakeSwapFeature.sol:387]:
```suggestion
                    case 4 {
```
and so forth?

File: contracts/zero-ex/contracts/src/features/PancakeSwapFeature.sol [hunk 2]
```diff
@@ -47,12 +47,30 @@ contract PancakeSwapFeature is
     uint256 constant private FF_BAKERYSWAP_FACTORY = 0xff01bf7c66c6bd861915cdaae475042d3c4bae16a70000000000000000000000;
```
Comment (dorothy-zbornak) [inline contracts/zero-ex/contracts/src/features/PancakeSwapFeature.sol:47]: Don't forget to bump up the minor number in `FEATURE_VERSION` in this file

**Review by dorothy-zbornak — APPROVED** [review]
1 little issue and changelog plz but otherwise fantastic work!
:+1: :eye: :lips: :eye: :+1:

File: contracts/zero-ex/contracts/src/features/PancakeSwapFeature.sol (hunk 1 above)
Comment (romainbutteaud) [reply contracts/zero-ex/contracts/src/features/PancakeSwapFeature.sol:387]: 😅 good catch

**Pull request merged**